README.md

```diff
@@ -14,17 +14,138 @@ comes from a linked-data triple store via an API. ActiveRecord
+    StrictHostKeyChecking no
+    UserKnownHostsFile /dev/null
+
+Host 10.10.10.* hmlr_*
+  ProxyJump hmlr-bastion
+  IdentityFile  ~/.ssh/lr.pem
+  User ec2-user
+  StrictHostKeyChecking no
+  UserKnownHostsFile /dev/null
+```
+
+Next we need to know the hostname of the service that's currently hosting the running
+service in the `dev` environment. This will typically be a name like `hmlr-dev-pres_071`,
+but the numerical suffix will change as the cluster is periodically updated by the ops
+team. There are two ways (at least) that we can get this information:
+
+- using [Sensu](https://sensu-hmlr.epimorphics.net/), using the **entities** tab, or
+- using the AWS console, under EC2 instances.
+
+In both cases, you will need credentials to log in to Sensu or AWS. See someone in the
+infrastructure team if you need credentials but don't have them.
+
+The remainder of this guide will assume `hmlr_dev_pres_071`, which is correct as of the
+time of writing (but may not be by the time you are reading this!)
+
+Check that you can access a server by directly connecting via ssh:
+
+```sh
+$ ssh hmlr_dev_pres_071
+Warning: Permanently added 'hmlr-bastion.epimorphics.net' (ECDSA) to the list of known hosts.
+**Warning**: this is a private system operated by Epimorphics Ltd
+Warning: Permanently added 'hmlr_dev_pres_071' (ECDSA) to the list of known hosts.
+**Warning**: this is a private system operated by Epimorphics Ltd
+$
+```
+
+Assuming this succeeds, you can set up an ssh tunnel to map the port where the data
+API is running on the remote machine to a convenient port on your computer. The local
+port you choose is up to you, but a good choice is port 8080. The data service runs
+on port 8081 on the remote service, so the tunnel command is:
+
+```sh
+$ ssh -f hmlr_dev_pres_071 -L 8080:localhost:8081 -N
+Warning: Permanently added 'hmlr-bastion.epimorphics.net,3.251.30.65' (ECDSA) to the list of known hosts.
+**Warning**: this is a private system operated by Epimorphics Ltd
+Warning: Permanently added 'hmlr_dev_pres_071' (ECDSA) to the list of known hosts.
+**Warning**: this is a private system operated by Epimorphics Ltd
+```
+
+If this succeeds, you should be able to see the open port with `lsof`:
+
+```sh
+$ lsof -i :8080
+COMMAND   PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
+ssh     71358  ian    3u  IPv6 766085      0t0  TCP ip6-localhost:http-alt (LISTEN)
+ssh     71358  ian    6u  IPv4 766086      0t0  TCP localhost:http-alt (LISTEN)
+```
+
+**Note:** the SSH tunnel will now be running as a background process. You should remember
+to stop it when you no longer require access to the API. A command such as `kill` on
+the process ID (PID) listed in the output from `lsof` will suffice:
+
+```sh
+$ kill -HUP 71358
+```
+
+Assuming there are no errors, you will see no output from this command.
+
+Having set up the SSH tunnel, you can then use `localhost:8080` as the data
+service URL to give to the application via the environment:
+
+```sh
```

Comment:
This logging is non-compliant to the requirement fro json. We should not be preserving it within the readme.